android/sdk/src/main/java/com/taobao/weex/WXSDKInstance.java

```diff
@@ -454,6 +454,7 @@ public void setUseScroller(boolean use) {
   public void setInstanceViewPortWidth(int instanceViewPortWidth) {
     this.mInstanceViewPortWidth = instanceViewPortWidth;
     this.mAutoAdjustDeviceWidth = false;
+    WXBridgeManager.getInstance().setViewPortWidth(getInstanceId(),mInstanceViewPortWidth);
```

Comment:
There is no need to call c++ code if this `setInstanceViewPortWidth` is invoked by WXMetaModule.setViewport